setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="tsutils",
-    version="1.1.0",
+    version="1.2.0",
```

Comment:
This is technically a breaking change if you were doing individual function imports, so bump version to 2.1.0

Reply:
wait i totally mean 2.0.0